src/app/(sidebar)/endpoints/[[...pages]]/page.tsx

```diff
@@ -864,6 +880,24 @@ export default function Endpoints() {
     );
   };
 
+  const renderPostAsyncTxResponseMessage = () => {
+    if (pathname === Routes.ENDPOINTS_TRANSACTIONS_POST_ASYNC && endpointData) {
+      if (endpointData.json.tx_status === "PENDING" && endpointData.json.hash) {
+        const href = buildEndpointHref(Routes.ENDPOINTS_TRANSACTIONS_SINGLE, {
+          transaction: endpointData.json.hash,
```

Comment:
when a user clicks `Check the async transaction’s status here`, should the hash get also passed to the single transaction page?

Reply:
@jeesunikim , yes, the Transaction Hash field should be pre-populated. Does it not do it for you? It seems fine for me.

https://github.com/user-attachments/assets/636a0666-473a-4e96-9fe0-689194b4f404

Reply:
oh it does! i was hallucinating maybe